tools/clang/lib/SPIRV/EmitVisitor.cpp

```diff
@@ -2438,9 +2438,9 @@ uint32_t EmitTypeHandler::emitType(const SpirvType *type) {
     // Emit Block or BufferBlock decorations if necessary.
     auto interfaceType = structType->getInterfaceType();
     if (interfaceType == StructInterfaceType::StorageBuffer)
-      // BufferBlock decoration is deprecated in Vulkan 1.2 and later.
+      // BufferBlock decoration is deprecated in SPIR-V 1.4 and later.
```

Comment:
To match the spec explicitly: "BufferBlock decoration is missing after SPIR-V 1.3." (since it was probably deprecated earlier, but still valid).

Reply:
I can drive-by update the original comment if desired, yes.

Reply:
Fwiw we could also follow `spirv-val` and use `requires SPIR-V version 1.3 or earlier`?

Reply:
Sure, both "is missing/removed after SPIR-V 1.3" or "requires SPIR-V 1.3 or earlier" are fine.

Reply:
Thanks, updated.